modisco

```diff
@@ -78,6 +81,61 @@ The options are as follows:
 )
 meme_parser.add_argument("-o", "--output", type=str, help="The path to the output file.")
 
+seqlet_bed_parser = subparsers.add_parser("seqlet-bed", help="Create a BED file of seqlets from a modisco results file.")
+seqlet_bed_parser.add_argument("-i", "--h5py", type=str, required=True,
+	help="An HDF5 file containing the output from modiscolite.")
+seqlet_bed_parser.add_argument("-o", "--output", type=str, required=True,
+	help="The path to the output file.")
+seqlet_bed_parser.add_argument("-p", "--peaksfile", type=str, required=True,
+	help="The path to the peaks file. This is to compute the absolute start and\
+end positions of the seqlets within a reference genome, as well as the chroms.")
+seqlet_bed_parser.add_argument("-c", "--chroms", type=str, required=True,
+	help="""A comma-delimited list of chromosomes, or '*', denoting which
+chromosomes to process. Should be the same set of chromosomes used during
+interpretation. Examples: 'chr1,chr2,chrX' || '*' || '1,2,X'.""")
```

Comment:
Does `1,2,X` convert to `chr1, chr2, chrX` or is it another example of passing in names? People will frequently just prepend 'chr' to these but I think people should be explicit about what the exact name is since sometimes people also write `1` instead of `chr1`

Reply:
Should it be 1,2,3 or chr1,chr2,chr3?

Reply:
Done- edited to that user must pass in exact strings, and '*' leaves peaks file intact